source/reference/minio-mc/mc-tag.rst

```diff
@@ -20,6 +20,8 @@ The :mc:`mc tag` command adds, removes, and lists tags associated to a bucket or
 
 .. end-mc-tag-desc
 
+An object can have up to 10 custom tags.
```

Comment:
🤔 could be nice to have the three mentions of `10 tags` use identical wording. Although I only even noticed because the diff makes them visible near each other. Feel free to ignore this comment.